src/sql/workbench/contrib/executionPlan/browser/executionPlanPropertiesViewBase.ts

```diff
@@ -262,6 +269,16 @@ export abstract class ExecutionPlanPropertiesViewBase extends Disposable impleme
 		this._tableComponent.columns = columns;
 	}
 
+	protected setExpansionModeForAllCollapsiblePropertyRows(tableRows: Slick.SlickData[], expand: boolean): void {
+		tableRows.forEach(row => {
+			if (row.treeGridChildren && row.treeGridChildren.length > 0) {
+				row.expanded = expand;
```

Comment:
This isn't setting the expansion "mode" - but rather the actual expanded state. So I'd suggest renaming this to `setPropertyRowsExpanded` (and probably add a doc comment clarifying that this is a recursive expansion)